packages/valory/skills/apy_estimation_chained_abci/skill.yaml

```diff
@@ -184,7 +183,7 @@ models:
       response_key: data
       response_type: dict
       retries: 5
-      url: https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2
+      url: https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2
```

Comment:
Where does this url comes from?

Reply:
Copied it from the [official page](https://docs.uniswap.org/sdk/subgraph/subgraph-data).